### ISSUE TITLE
目前机器人团本只有naxx的策略，能否增加ICC的策略。

### ISSUE THREAD

**Issue by oiuv** [liyunfan1223/mod-playerbots#162]
目前机器人团本只有naxx的策略，能否增加ICC的策略。

**Comment by liyunfan1223** [comment]
Yeah it's a major goal of the project, but it takes a lot of effort to achieve. I'm not sure when to start working on it. You can try it if interested.

**Comment by Dreathean** [comment]
It is tough to argue a priority for that aspect when there are still many primary systems needing work on/fixing and for now we are just extremely fortunate that Liyunfan has the interest, knowledge, and willpower to work on these. Ultimately unless we can get more people to help it all comes down to what they are interested in doing.

**Comment by oiuv** [comment]
> Yeah it's a major goal of the project, but it takes a lot of effort to achieve. I'm not sure when to start working on it. You can try it if interested.

我不太擅长这方面的开发，一方面在编程语言上我对C++不是足够擅长，另一方面我对这些功能开发的逻辑也不够理解，心有余而力不足。现在我在我的体验服上一直是保持机器人最新版，也是用我的玩家小伙伴做测试，也只能根据玩家的测试反馈更新优化一些小功能。
